api/proto/metrics/metrics.proto

```diff
@@ -123,12 +158,19 @@ message deviceMetric {
   repeated MetricItem metricItems = 8;
   uint64 runtimeStorageOverheadMB = 9;     // In MB
   uint64 appRunTimeStorageMB = 10;         // In MB
+
+  // systemServicesMemoryMB - OBSOLETE. Will no longer be used.
+  // Currently represents whole usage for Dom0
   memoryMetric systemServicesMemoryMB = 11;  // In MB
+
   logMetric log = 12;
   repeated CipherMetric cipher = 13;  // Object decryption
   AclMetric acl = 14;
   newlogMetric newlog = 15;
   zedboxStats zedbox = 16;
+
+  // memory_v2 - Memory metrics.
+  DeviceMemoryMetric memory_v2 = 17;
```

Comment:
Can we find a different name than v2? 
deviceMemory can be the name of the field.

Reply:
Done.